docs/docs/how_to/pebblo_retrieval_qa.ipynb

```diff
@@ -0,0 +1,488 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3ce451e9-f8f1-4f27-8c6b-4a93a406d504",
+   "metadata": {},
+   "source": [
+    "# Identity-enabled RAG using PebbloRetrievalQA\n",
+    "\n",
+    "> PebbloRetrievalQA is a Retrieval chain with Identity & Semantic Enforcement for question-answering\n",
+    "against a vector database.\n",
+    "\n",
+    "This notebook covers how to retrieve documents using Identity & Semantic Enforcement (Deny Topics/Entities).\n",
+    "\n",
+    "### Steps:\n",
+    "\n",
+    "1. **Loading Documents:**\n",
+    "We will load documents with authorization and semantic metadata into an in-memory Qdrant vectorstore. This vectorstore will be used as a retriever in PebbloRetrievalQA.\n",
```

Comment:
Add a note here about using `PebbloSafeLoader` to load docs with auth & semantic metadata. We want to convey that this chain is best used with PebbloSafeLoader as the counterpart on the ingestion side.